hw/drivers/sensors/dps368/pkg.yml

```diff
@@ -0,0 +1,46 @@
+#Copyright 2019 Infineon Technologies AG
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+#
+
+pkg.name: "hw/drivers/sensors/dps368"
+pkg.description: Driver for Infineon DPS368 pressure sensor
+pkg.author: "Saumitra Chafekar <saumitra.chafekar@infineon.com>"
```

Comment:
This should be changed to Apache Mynewt. This is as per guidelines by Apache, there might have been some remnants in the driver code which we need to cleanup but the author should be Apache.

Reply:
Earlier as per my reference from existing driver code base, authors name other than Apache was found and hence simply followed the same as guideline. 
This has been fixed now.